README.md

```diff
@@ -248,291 +178,232 @@ Logging in to balena-staging.com
 
 $ cd ~/balenaProduction
-$ balena apps
-ID      APP NAME         DEVICE TYPE      ONLINE DEVICES DEVICE COUNT
-54321   artik530         artik533s        0              0
-54322   bob              intel-nuc        0              0
-54323   bbbtest          beaglebone-black 0              0
+$ balena fleets
+ Id      App name       Slug                 Device type     Device count Online devices
+ ─────── ────────────── ──────────────────── ─────────────── ──────────── ──────────────
+ 1913403 balena-health  ryanh/balena-health  raspberrypi4-64 1            0
+ 1922430 balena-pricing ryanh/balena-pricing raspberrypi4-64 1            0
+ 1962591 notice-board   ryanh/notice-board   raspberrypi3-64 1            0
 ```
 
-For the purposes of this document, we have chosen `'.'` (the current working
-directory) as the value for the `dataDirectory` property in the `balenarc.yml`
-file, but you could of course choose different directories such as, for example,
-`/home/joe/.balena/production` and `/home/joe/.balena/staging`. If this property
-is not specified, the default data directory is `.balena` or `_balena`
-(respectively on Linux/macOS and Windows) in the user's home directory.
+For the purposes of this document, we have chosen `'.'` (the current working directory) as the value for the `dataDirectory` property in the `balenarc.yml` file, but you could of course choose different directories such as `/home/ryanh/.balena/production` and `/home/ryanh/.balena/staging`. If this property is not specified, the default data directory is `.balena` or `_balena` (respectively on Linux/macOS and Windows) in the user's home directory.
 
-> Watch out! The tilde character (`'~'`) is not resolved to the homedir when
-> used inside a `balenarc.yml` file. Use the full directory such as `/home/joe/`
-> instead of the tilde.
+> Watch out! The tilde character (`'~'`) is not resolved to the homedir when used inside a `balenarc.yml` file. Use the full directory such as `/home/ryanh/` instead of the tilde.
 
-It is also worth noting that the CLI accepts the `BALENARC_DATA_DIRECTORY`
-environment variable as an alternative to the `dataDirectory` property in the
-`balenarc.yml` file. For example, the following command would allow the CLI to
-use the staging settings and token even if the current working directory was not
-`~/balenaStaging`:
+It is also worth noting that the CLI accepts the `BALENARC_DATA_DIRECTORY` environment variable as an alternative to the `dataDirectory` property in the `balenarc.yml` file. For example, the following command would allow the CLI to use the staging settings and token even if the current working directory was not `~/balenaStaging`:
 
 ```shell
 $ cd /tmp
 $ BALENARC_BALENA_URL=balena-staging.com \
   BALENARC_DATA_DIRECTORY=~/balenaStaging \
   balena whoami
+== ACCOUNT INFORMATION
+USERNAME: ryanh
+EMAIL:    ryanh@balena.io
+URL:      balena-cloud.com
```

Comment:
Is that the correct output? Above it the environment the command is targeting is the staging environment. 
Let me know, I could be wrong. 
```suggestion
URL:      balena-staging.com
```

Reply:
@vipulgupta2048 Good catch! This output shouldn't even be there I don't think.